package.json

```diff
@@ -35,7 +35,8 @@
     "eol": "0.9.1",
     "lodash": "^4.0.0",
     "promise": "^8.0.0",
-    "yargs": "^11.0.0"
+    "yargs": "^11.0.0",
+    "fs-extra": "^6.0.1"
```

Comment:
v6 drops node 4, so that might be related to the test failures. bumping this back down to 5 may give you better luck. can you also reorder it while you're at it?